src/Illuminate/Queue/Console/ClearCommand.php

```diff
@@ -16,9 +18,7 @@ class ClearCommand extends Command
      *
      * @var string
      */
-    protected $signature = 'queue:clear
-                            {connection? : The name of the queue connection to clear}
-                            {--queue= : The name of the queue to clear}';
+    protected $signature = 'queue:clear';
```

Comment:
Just add a third line to the signature with `{--force : Force the operation to run when in production}`. There's really no need to override the argument/options methods.

Reply:
@paras-malhotra I know ;)

However, it's not in line with the other clear commands
> I was also so free to update the command to be in line with the other clear commands.

If Taylor doesn't want this, he will probably tell me ;)

Reply:
Yeah, I added this because its quite unusual to clear queues in production (unlike clearing cache, cached routes, etc.). Sure, let's wait for Taylor's feedback :thumbsup:

Reply:
To me, it doesn't make sense to add the `ConfirmableTrait` without being able to override it.

Reply:
Yeah, I agree with that. I do think we need to add the force option. I'm just saying it's best for the trait to stay there.